SAFARI/BCAGPA.js

```diff
@@ -0,0 +1,164 @@
+// ==UserScript==
+// @name        BCA GPA
+// @description GPA Calculator for BCA
+// @match       https://ps001.bergen.org/*
+// ==/UserScript==
+
+if (document.URL === "https://ps001.bergen.org/guardian/home.html") {
```

Comment:
Instead of having an entirely separate file that would also have duplicate updates every time something in the base extension is changed, can we just add this to `src/main.js`?

Reply:
Main.js has been updated to contain this